build.sh

```diff
@@ -1,5 +1,23 @@
 #! /bin/sh
 
+which git 2> /dev/null > /dev/null
```

Comment:
You could shorten this to `which git > /dev/null 2>&1` I think.

Reply:
Agreed.